### ISSUE TITLE
Unclear instructions for setting up data structure

### ISSUE THREAD

**Issue by alexgriff** [learn-co-curriculum/advanced-hashes-hashketball#30]
_** I accidently first raised this issue on the learn-co-students repo, closed that issue and reposting here_ 

In the instructions for setting up the initial data structure it reads: 

> The `:players` key points to a hash of players whose names (as strings) are the keys to a hash containing their stats.

While it is possible to solve the lab by setting it up that way the solution branch has the `:players` key point to an **array**.  An array of hashes like this:

```
:away => 
{ :team_name => "Brooklyn Nets", 
  :colors => ["Black", "White"], 
  :players => [ 
    {:player_name => "Alan Anderson", :number => 0, ... }, 
    {:player_name => "Reggie Evans", :number => 30, ... }, ... 
  ]
}
```

Should the readme be updated to reflect the solution or the solution changed to reflect the readme?

To me it makes more sense that the key `:players` points to a list, an array. That seems the more intuitive data structure for the question "give me all the players on this team".

**Comment by realAndrewCohn** [comment]
came across this same issue when doing a review for students. I'm going to update the readme and submit a PR

**Comment by realAndrewCohn** [comment]
This issue has been resolved in #34 The solution branch now reflects the instructions in the readme. Given that the goal of this lab is to navigate through hashes, `players` should remain a hash where the keys are the player names. We want to be able to find a given player in constant time assuming we know their name.
